OpenEdXMobile/build.gradle

```diff
@@ -310,7 +314,21 @@ android {
                 resValue "string", "io.fabric.ApiKey", fabric_key
                 resValue "bool", "com.crashlytics.RequireBuildId", "false"
             }
+            def kits = fabric.get('KITS')
```

Comment:
Branch is independent of Fabric, so it shouldn't be under check of Fabric enability.

Reply:
Good point!